server/src/main/java/org/apache/druid/server/coordinator/helper/NewestSegmentFirstIterator.java

```diff
@@ -201,7 +208,7 @@ private void updateQueue(String dataSourceName, DataSourceCompactionConfig confi
               .filter(holder -> {
                 final List<PartitionChunk<DataSegment>> chunks = Lists.newArrayList(holder.getObject().iterator());
                 final long partitionBytes = chunks.stream().mapToLong(chunk -> chunk.getObject().getSize()).sum();
-                return chunks.size() > 1
+                return chunks.size() > 0
```

Comment:
Prefer `!chunks.isEmpty()` (similar to the change you made on line 185)

Reply:
Changed.